recipes/plothist/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "plothist" %}
+{% set version = "1.2.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/plothist-{{ version }}.tar.gz
+  sha256: b4606d9ff7db3cc1a7f6f5d5440fd959e1d1292818910cc0d142b23e65e4de76
+
+build:
+  entry_points:
+    - install_latin_modern_fonts = plothist.scripts.install_latin_modern_fonts:install_latin_modern_fonts
+    - plothist_make_examples = plothist.scripts.make_examples:make_examples
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - flit-core >=3.2,<4
+    - pip
+  run:
+    - python >=3.7
+    - boost-histogram >=1.4.0,<1.5.dev0
+    - numpy >=1.14.5
+    - matplotlib-base >=3.0
+    - pyyaml >=5.3.1
+    - scipy >=1.6.0
+
+test:
+  imports:
+    - plothist
+  commands:
+    - pip check
+    - install_latin_modern_fonts --help
+    - plothist_make_examples --help
+  requires:
+    - pip
+
+about:
+  home: https://github.com/cyrraz/plothist
+  summary: Plot histograms in a scalable way and a beautiful style.
+  license: BSD-3-Clause
+  license_file: LICENSE
+  doc_url: https://plothist.readthedocs.io/
+  dev_url: https://github.com/cyrraz/plothist
+
+extra:
+  recipe-maintainers:
+    - cyrraz
+    - 0ctagon
```

Comment:
@cyrraz @0ctagon Can you add a comment as a post on the PR if you are willing to be a maintainer on this conda-forge recipe (required before a conda-forge maintainer can review this).

Reply:
Thank you for the PR!
Is it possible to remove @cyrraz, as I'm the only long term maintainer currently?

Reply:
Done. The conda-forge linter requires you to comment as a standalone PR comment with something like "I am willing to be listed as a maintainer."